specification/communication/resource-manager/readme.md

```diff
@@ -53,6 +53,7 @@ swagger-to-sdk:
   - repo: azure-sdk-for-python
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
     after_scripts:
       - bundle install && rake arm:regen_all_profiles['azure_mgmt_vsonline']
```

Comment:
JS SDK doesn't have after scripts, you should remove the after scripts or put it right after the repo that needs it.

Reply:
Good catch. That after_script is not needed. I'll remove it.